rust/datafusion/src/execution/context.rs

```diff
@@ -310,16 +324,25 @@ impl ExecutionContext {
         name: impl Into<String>,
         catalog: Arc<dyn CatalogProvider>,
     ) -> Option<Arc<dyn CatalogProvider>> {
-        self.state
-            .lock()
-            .unwrap()
-            .catalogs
-            .insert(name.into(), catalog)
+        let name = name.into();
+
+        let state = self.state.lock().unwrap();
+        println!("Registering catalog for {}", name);
```

Comment:
Leftover debugging?

Reply:
whoops -- good call -- will remove.

Reply:
might be a good one for a `debug!` line actually.